plugins/kubernetes-backend/src/service/KubernetesProxy.ts

```diff
@@ -172,8 +172,9 @@ export class KubernetesProxy {
             )?.toString(),
           };
 
-          const authHeader = req.header(HEADER_KUBERNETES_AUTH);
-          if (authHeader) {
+          const authHeader =
+            req.headers[HEADER_KUBERNETES_AUTH.toLocaleLowerCase()];
```

Comment:
```suggestion
            req.headers[HEADER_KUBERNETES_AUTH.toLocaleLowerCase('en-US')];
```

Should always be explicit about the locale to avoid issues in certain environments

Reply:
I changed to use toLowerCase() instead

Reply:
Please don't, that's just as bad :) `.toLocaleLowerCase('en-US')` is what we use in cases like this one across the board. For context, what you have will cause issues in for example the Turkish locale where lover case I is not i, but ı.

Reply:
I found this in the same file:    
https://github.com/backstage/backstage/blob/master/plugins/kubernetes-backend/src/service/KubernetesProxy.ts#L232 
`const clusterName = req.headers[HEADER_KUBERNETES_CLUSTER.toLowerCase()];`